src/common/components/table/table-examples/BlocksTable.tsx

```diff
@@ -83,7 +83,10 @@ export function BlocksTable() {
     pageSize: 10,
   });
 
-  const response = useBurnBlocks(pagination.pageSize, pagination.pageIndex * pagination.pageSize);
+  const response = useBurnBlocksInfinite(
```

Comment:
I don't think we need to use infinite here as each request is discreet and no longer relies on getnextpageparam

Reply:
Make sense, that was just renaming the function though, I suggest we make this refactor in a separate pr.